popup.js

```diff
@@ -0,0 +1,64 @@
+// use DOMContentLoaded to wait for loading HTML 
+document.addEventListener("DOMContentLoaded", function(){
+    var selected_folder_text_field = document.getElementById('selected_folder')
+    var selected_folder_set_button = document.getElementById('selected_folder_button')
+    
+    // make text fields editable if they are tapped
+    selected_folder_text_field.addEventListener("click", function() {
+        console.log("selected folder text field is clicked!")
+        selected_folder_text_field.contentEditable = true
+    });
```

Comment:
The text field is always empty when the popup is opened at the moment. The selected folder name stored in chrome storage should be loaded when the popup is opened.